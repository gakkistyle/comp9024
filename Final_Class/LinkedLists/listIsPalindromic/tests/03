9 4 7 1 1 7 4 9
