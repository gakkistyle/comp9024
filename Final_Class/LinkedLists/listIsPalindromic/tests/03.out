Enter list: listIsPalindromic returned TRUE
